lib/ansible/modules/cloud/openstack/os_server_facts.py

```diff
@@ -47,6 +47,10 @@
           of additional API calls.
      required: false
      default: false
+   filters:
+     description:
+        - restrict results to servers matching a dictionary of
+          filters (e.g., {vm_state: 'active'})
```

Comment:
This line is breaking the YAML, I think  you need something closer to 

`filters, for example: C({vm_state: 'active'}).`